src/config.env.ts

```diff
@@ -93,7 +97,8 @@ const config: { [env: string]: IServerOptions } = {
  *
  * default: development
  */
-const envConfig = config[process.env.NODE_ENV || 'development'] || config.development;
+const envConfig = config[process.env['NODE' + '_ENV'] || 'development'] || config.development;
```

Comment:
What is the different between:
- `process.env.NODE_ENV`
- `process.env['NODE_ENV']`
- `process.env['NODE' + '_ENV']`?